assets/questions/triangle.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Run-Length Encoding.
+ */
+
+globalData.questions['triangle'] = {  // eslint-disable-line dot-notation
+  title: 'Run-Length Encoding',
+  starterCode: {
+    python:
+`def transform(str):
+    lines=str.split('<')
+    (n,e)=lines[0].split('.')
+    n=int(n)
+    e=int(e)
+    graph=[[False for _ in range(n)] for _ in range(n)]
+    
+    for i in range(e):
+        (u,v)=lines[i+1].split('.')
+        u=int(u)
+        v=int(v)
+        graph[u][v]=True
+        graph[v][u]=True
+    return graph
+def countTriangle(str):
+    graph=transform(str)
+    return None
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def skipEncodingAtEndOfString(cls, word):
```

Comment:
You'll need some auxiliaryCode if you have any BuggyOutputTests, which you should have.